libs/ui-components/yarn.lock

```diff
@@ -0,0 +1,363 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
why did you need to run `yarn install` in this lib folder? did you have any error? or can we remove this file?

Reply:
I don't remember it, so just a mistake I suppose. I'll delete it